typegraph/python/typegraph/runtimes/deno.py

```diff
@@ -132,10 +131,17 @@ def __attrs_post_init__(self):
             if self.code is not None:
                 raise Exception("you must only give either source file or source code")
 
-            from typegraph.graph.typegraph import get_absolute_path
+            # from typegraph.graph.typegraph import get_absolute_path
+
+            # depth 0: typegraph.py (definition)
```

Comment:
not always true, for instance if an importer is used into another file?

Reply:
Definitely, this is why I dropped the inspect approach.